src/circular-progress.jsx

```diff
@@ -144,13 +144,13 @@ const CircularProgress = React.createClass({
   _rotateWrapper(wrapper) {
     if (this.props.mode !== 'indeterminate') return;
 
-    AutoPrefix.set(wrapper.style, 'transform', 'rotate(0deg)');
```

Comment:
Any reason why we can't call `muiTheme.prefix()` rather than calling it on the imported `auto-prefix` from here?